src/AdvApp2Var.cpp

```diff
@@ -268,7 +268,7 @@ PYBIND11_MODULE(AdvApp2Var, mod) {
 	cls_AdvApp2Var_ApproxAFunc2Var.def("UFrontError", (Standard_Real (AdvApp2Var_ApproxAFunc2Var::*)(const Standard_Integer, const Standard_Integer) const ) &AdvApp2Var_ApproxAFunc2Var::UFrontError, "returns the error max of the BSplineSurface of range Index on a UFrontier", py::arg("Dimension"), py::arg("Index"));
 	cls_AdvApp2Var_ApproxAFunc2Var.def("VFrontError", (Standard_Real (AdvApp2Var_ApproxAFunc2Var::*)(const Standard_Integer, const Standard_Integer) const ) &AdvApp2Var_ApproxAFunc2Var::VFrontError, "returns the error max of the BSplineSurface of range Index on a VFrontier", py::arg("Dimension"), py::arg("Index"));
 	cls_AdvApp2Var_ApproxAFunc2Var.def("CritError", (Standard_Real (AdvApp2Var_ApproxAFunc2Var::*)(const Standard_Integer, const Standard_Integer) const ) &AdvApp2Var_ApproxAFunc2Var::CritError, "None", py::arg("Dimension"), py::arg("Index"));
-	// FIXME cls_AdvApp2Var_ApproxAFunc2Var.def("Dump", (void (AdvApp2Var_ApproxAFunc2Var::*)(Standard_OStream &) const ) &AdvApp2Var_ApproxAFunc2Var::Dump, "Prints on the stream o informations on the current state of the object.", py::arg("o"));
+	// FIXME cls_AdvApp2Var_ApproxAFunc2Var.def("Dump", (void (AdvApp2Var_ApproxAFunc2Var::*)(Standard_OStream &) const ) &AdvApp2Var_ApproxAFunc2Var::Dump, "Prints on the stream o information on the current state of the object.", py::arg("o"));
```

Comment:
`Prints on the stream o information on current state of the object` This should be re-written